### PR TITLE
Tel 157 tenant authentication

### PR DESCRIPTION
**Pull request by mrcpvn**

### CODE REVIEW

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 31358ddf0832a2bb575602852f83aeaa9fed0b0b
* Result: FAILED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=be2cc424-3b95-4f62-9255-594875b77c55%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/WWTelematics.TenantService/Messages/TenantAuthenticationRequest.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace WWTelematics.TenantService.Messages
```
Comment (davidw-foxtrax) [inline src/WWTelematics.TenantService/Messages/TenantAuthenticationRequest.cs:3]: Can you please put in this in the Models namespace?

Comment (mrcpvn) [reply src/WWTelematics.TenantService/Messages/TenantAuthenticationRequest.cs]: messages namespace moved into models

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 93bf2a24137216f7e2b6e4dd405f551594ddbf11
* Result: SUCCEEDED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=b0aa6305-d168-4760-88de-aeaff4b14027%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/WWTelematics.TenantService/Messages/TenantAuthenticationRequest.cs (hunk 1 above)
Comment (mrcpvn) [reply src/WWTelematics.TenantService/Messages/TenantAuthenticationRequest.cs]: https://github.com/WorkWave/WWTelematics.ClientService/pull/29#discussion_r379640534

I created the Messages namespace to be consistent with the Client service

Comment (davidw-foxtrax) [reply src/WWTelematics.TenantService/Messages/TenantAuthenticationRequest.cs]: Ack, I missed that one when I updated Clients; that one should be updated, too.  Now that the Entity classes have been properly renamed, we should use Models for data models.

Comment (mrcpvn) [reply src/WWTelematics.TenantService/Messages/TenantAuthenticationRequest.cs]: removed Messages namespace and moved the authentication model to Models

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 4419bdf1ef886972682a293eb3e212e5967c8388
* Result: SUCCEEDED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=a09a2a8f-655f-40d0-9434-2a44dae346a5%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 44c7cd3529d708ae21fbc5d03fa3a3c0204ec35c
* Result: SUCCEEDED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f565c099-46e5-4a6b-bda1-4877dc173ac1%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**